app/templates/login.hbs

```diff
@@ -0,0 +1,141 @@
+<AuApp>
```

Comment:
I have to double check what this outputs, but I think this isn't  needed since we already have it in the application.hbs template. Same with the main container.

Reply:
Yea, they aren't needed here.